src/Containers/Chart.tsx

```diff
@@ -159,11 +159,13 @@ const CustomTooltip: React.FC<CustomTooltipProps> = ({
     payload,
     ...props
 }): React.ReactElement => {
+    const formatValue = (value) => value.toFixed(2);
+
     if (active) {
         return (
             <CustomTooltipBox active={active} payload={payload} {...props}>
                 <strong>
-                    $<Big>{`${payload[0].value.toFixed(2)}`}</Big>
+                    $<Big>{`${formatValue(payload[0].value)}`}</Big>
```

Comment:
you need to replace this entire line with {getFormattedHoverValue()}

Reply:
Ok, thought would not work but it does. I fixed it in next commit